code/game/gamemodes/game_mode.dm

```diff
@@ -120,15 +163,39 @@
 		replacementmode.make_antag_chance(character)
 	return
 
+/datum/game_mode/proc/make_special_antag_chance(mob/living/character)
+	if(character.mind.antag_datums)
+		if(character.mind.antag_datums.len > 0)
+			return
+	//Check if they are banned
+	if(is_banned_from(character.ckey, list(ROLE_SPECIAL)) || QDELETED(character))
+		return
+	for(var/datum/special_role/subantag in active_specials)
+		if(!subantag.latejoin_allowed)
+			continue
+		if(subantag.spawn_mode == "midround")
+			continue
+		var/count = 0
+		for(var/mob/living/M in GLOB.mob_list)
+			if(!M.mind)
+				continue
+			if(!is_special_type(M, subantag.attached_antag_datum))
+				continue
+			count ++
+		if(count >= subantag.max_amount)
+			continue

```

Comment:
Not `break`?

Reply:
This should be continue, as if there are too many of the antag it is checking if it can make, it moves on to checking the next antag to see if they can be made. Otherwise, if antag type A has too many members, then antag type B will not be created either, even if it should.